cmds/modules/provisiond/main.go

```diff
@@ -147,10 +153,28 @@ func action(cli *cli.Context) error {
 	// keep track of resource units reserved and amount of workloads provisionned
 
 	// to store reservation locally on the node
-	store, err := storage.NewFSStore(filepath.Join(rootDir, "workloads"))
+	store, err := storage.New(filepath.Join(rootDir, boltStorageDB))
 	if err != nil {
 		return errors.Wrap(err, "failed to create local reservation store")
 	}
+	defer store.Close()
+	// we check if the old fs storage still exists
+	fsStoragePath := filepath.Join(rootDir, fsStorageDB)
+	if _, err := os.Stat(fsStoragePath); err == nil {
+		// if it does we need to migrate this storage to new bolt storage
+		fs, err := fsStorage.NewFSStore(fsStoragePath)
+		if err != nil {
+			return err
+		}
+
+		if err := storageMigration(store, fs); err != nil {
+			log.Error().Err(err).Msg("storage migration failed")
```

Comment:
I was thinking about this during the implementation. But I didn't want to over engineer it since main-net is still not actively used. May be another simpler solution is to delete deployments that has been succesfully migrated. 

The reason i didn't want to do this, is that if failed deployment were skipped, the migration won't be tried again unless the node has been rebooted or on a new update during this time the deploymend (the un-migrated one) will be un-available. calls to deployment.get will return not found.

I will see if i can keep the failed one to be tried again.